bundlewrap/items/files.py

```diff
@@ -341,19 +341,31 @@ def sdict(self):
             }
 
     def display_dicts(self, cdict, sdict, keys):
-        if (
-            'content_hash' in keys and
-            self.attributes['content_type'] not in ('base64', 'binary') and
-            sdict['size'] < DIFF_MAX_FILE_SIZE and
-            len(self.content) < DIFF_MAX_FILE_SIZE and
-            PathInfo(self.node, self.name).is_text_file
-        ):
-            keys.remove('content_hash')
-            keys.append('content')
-            del cdict['content_hash']
-            del sdict['content_hash']
-            cdict['content'] = self.content
-            sdict['content'] = get_remote_file_contents(self.node, self.name)
+        if sdict is None and not self.attributes.get('delete', False):
+            if (
+                self.attributes['content_type'] not in ('base64', 'binary') and
+                len(self.content) < DIFF_MAX_FILE_SIZE
+            ):
+                keys.extend(['owner', 'group', 'mode', 'content'])
```

Comment:
Why are you _extending_ `keys` here? Is there any point in preserving what's in there?

```suggestion
                keys = ['owner', 'group', 'mode', 'content']
```

Reply:
Mainly because I wasn't sure what was passed from above and wanted to be compatible.

I agree that it makes more sense to overwrite it.